src/styleProps/textProps.ts

```diff
@@ -7,37 +7,39 @@ const textTransformMap = {
 	none: 'none',
 };
 
+type TextTransformKey = keyof typeof textTransformMap;
+
 export const textPropsDef = [
 	{
 		styleProp: 'bold',
 		properties: ['fontWeight'],
-		getValue: bold => (bold ? 'bold' : 'normal'),
+		getValue: (bold: unknown) => (bold ? 'bold' : 'normal'),
 	},
 	{
 		styleProp: 'fontSize',
 		properties: ['fontSize'],
-		getValue: (size, theme) =>
+		getValue: (size: string, theme: any) =>
 			size ? theme.globals.typeScale[size] : undefined,
 	},
 	{
 		styleProp: 'textAlign',
 		properties: ['textAlign'],
-		getValue: value => value,
+		getValue: (value: unknown) => value,
```

Comment:
I guess if we move the interface from the `Text` component into this module, we can use those types here, too.

Reply:
same as before seems a bigger fix with IStylePropsConfig